node-require/calculate.js

```diff
@@ -0,0 +1,20 @@
+const num1 = parseFloat(process.argv[2]);
+const operator = process.argv[3];
+const num2 = parseFloat(process.argv[4]);
+const add = require('./add');
+console.log('results:', add);
+const subtract = require('./subtract');
+console.log('results:', subtract);
+const multiply = require('./multiply');
+console.log('results:', multiply);
+const divide = require('./divide');
+console.log('results:', divide);
```

Comment:
I love the use of console logs to test your code, but once you have everything working remember to clean up your code by deleting the console logs.